File: Empresas/src/main/java/br/com/empresa/model/Empresa.java
package br.com.empresa.model;

import lombok.Data;

import javax.persistence.*;
import java.io.Serializable;

@Entity
@Data
@Table(name = "empresa", schema = "dbo")
public class Empresa implements Serializable {

    @Id
    @Column(name = "id")
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column(name = "cnpj")
    private String cnpj;

    @Column(name = "razao_social")
    private String razaoSocial;

    @Column(name = "nome_fantasia")
    private String nomeFantasia;

    @Column(name = "ativa")
    private boolean ativa;

    @OneToOne
    @JoinColumn(name = "arquivo", referencedColumnName = "id")
    private Arquivo foto;

}

File: src/database/models/empresa.ts
import { Model, DataTypes } from 'https://deno.land/x/denodb/mod.ts';

class Empresa extends Model {
  static fields = {
    _id: {
      primaryKey: true,
    },
    razaoSocial: {
      type: DataTypes.STRING
    },
    nomeFantasia: {
      type: DataTypes.STRING
    },
    cnpj: {
      type: DataTypes.STRING
    },
    foto: {
      type: DataTypes.STRING
    }
  };
}

export default Empresa;